client/rest-high-level/src/test/java/org/elasticsearch/client/ESRestHighLevelClientTestCase.java

```diff
@@ -78,20 +79,22 @@ protected static RestHighLevelClient highLevelClient() {
                                        AsyncMethod<Req, Resp> asyncMethod) throws IOException {
         return execute(request, syncMethod, asyncMethod, RequestOptions.DEFAULT);
     }
-    
+
     /**
      * Executes the provided request using either the sync method or its async variant, both provided as functions
      */
     protected static <Req, Resp> Resp execute(Req request, SyncMethod<Req, Resp> syncMethod,
                                        AsyncMethod<Req, Resp> asyncMethod, RequestOptions options) throws IOException {
-        if (randomBoolean()) {
+        final boolean async = Booleans.parseBoolean(System.getProperty("tests.rest.async", "false"));
```

Comment:
I think we can read this system property as a constant?
We either runs all tests with this set to false or true.

Reply:
yea def.